ffmpeg/transcoder.c

```diff
@@ -6,6 +6,7 @@
 
 #include <libavcodec/avcodec.h>
 #include <libavformat/avformat.h>
+#include <stdbool.h>
```

Comment:
I think we might need to include avfilter.h here - see [here](https://github.com/livepeer/lpms/commit/7f3f7c2881a63e56be53e116dbc030d4e3d3d9d9).

Without the include statements, I ran into compilation errors about missing references for `avfilter_register_lvpdnn` and `avfilter_remove_lvpdnn`.

Reply:
Updated install_ffmpeg.sh, added avfilter.h head file